packages/common/src/utils/validationUtils.test.ts

```diff
@@ -344,6 +345,59 @@ describe('Validate mandatory rule for uncertain datet', () => {
   })
 })
 
+describe('Validate mandatory rule for cause of death', () => {
+  const causeOfDeathElement: CertificateDataElement = {
+    id: '1',
+    parent: 'grundformu',
+    index: 3,
+    visible: true,
+    readOnly: false,
+    mandatory: true,
+    config: {
+      text: 'Dödsorsak',
+      description: 'Den terminala dödsorsaken',
+      type: ConfigTypes.UE_CAUSE_OF_DEATH,
+      id: 'dodsorsak',
+    },
+    value: {
+      type: CertificateDataValueType.CAUSE_OF_DEATH,
+      id: 'dodsorsak',
+      description: { text: 'description' },
+      debut: { date: '2020-02-02' },
+      specification: { code: 'KRONISK' },
+    },
+    validation: [
+      {
+        type: CertificateDataValidationType.MANDATORY_VALIDATION,
+        questionId: '1',
+        expression: '$dodsorsak',
+      },
+    ],
+    validationErrors: [],
+  }
```

Comment:
Instead of creating this object, we could use our helper function. For example: `fakeCauseOfDeathElement({ id: '1', value: { text: null } })[1]`